docs/source/app-dev/bindings-scala/index.rst

```diff
@@ -0,0 +1,144 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Scala bindings
+##############
+
+This tutorial provides a basic Scala programmer's introduction to working with Digital Asset distributed ledger,
+using the Scala programming language and the *Ledger API*.
+
+Introduction
+============
+
+The Scala bindings is a client implementation of the *Ledger API*. The Scala bindings library lets you write applications that connect to the Digital Asset distributed ledger using the Scala programming language.
+
+There are two main components:
+
+- Scala codegen
+    DAML to Scala code generator. Use this to generate Scala classes from DAML models. The generated Scala code provides a type safe way of creating contracts (:ref:`com.digitalasset.ledger.api.v1.createcommand`) and exercising contract choices (:ref:`com.digitalasset.ledger.api.v1.exercisecommand`).
+
+- Akka Streams-based API
+    The API that you use to send commands to the ledger and receive transactions back.
+
+In order to use the Scala bindings, you should be familiar with:
+
+- :doc:`DAML language </daml/reference/index>`
+- :doc:`Ledger API </app-dev/grpc/index>`
+- `Akka Streams API <https://doc.akka.io/docs/akka/current/stream/index.html>`_
+- `Scala programming language <https://www.scala-lang.org>`_
+- :ref:`assistant-manual-building-dars`
+
+Getting started with sbt
+========================
+
+Set up the following dependencies in your ``build.sbt``:
+
+.. literalinclude:: ./code-snippets/quickstart-scala/build.sbt
+   :start-after: // <doc-ref:dependencies>
+   :end-before: // </doc-ref:dependencies>
+
+We recommend separating generated code and application code into different modules. There are two modules in the example below:
+
+- ``scala-codegen``
+    This modules contains all generated Scala classes.
+- ``application``
+    This is the application code that makes use of the generated Scala classes.
+
+.. literalinclude:: ./code-snippets/quickstart-scala/build.sbt
+   :start-after: // <doc-ref:modules>
+   :end-before: // </doc-ref:modules>
+
+``scala-codegen`` module uses the following function to generate Scala classes from a DAR file.
+
+.. literalinclude:: ./code-snippets/quickstart-scala/build.sbt
+   :start-after: // <doc-ref:generate-scala>
+   :end-before: // </doc-ref:generate-scala>
+
+Here is the entire `build.sbt file <https://github.com/digital-asset/daml/blob/master/language-support/scala/examples/quickstart-scala/build.sbt>`_
+
+Generating Scala Code from Command Line
+=======================================
+
+The above example demonstrates how to use Scala codegen from **sbt**. You can also call Scala codegen directly
+from a command line. For information on how to build a **DAR** file from a **DAML** model see :ref:`assistant-manual-building-dars`.
```

Comment:
"For information on how to build a **DAR** file" comes completely out of the blue here - you haven't told anyone why they'd need to care about it at all! 

I'd suggest rewriting this as a set of instructions:

1. If you're not on the latest, upgrade the SDK
2. Download the codegen tool
3. Build the DAR file (this is the right place to link to instructions)
4. Run the codegend tool.

The reason for this is that you need to help people out: you're giving them a bunch of information but it's not in the logical order that they'd need to follow it in, so it makes it much tougher to use this doc.